api/security/security.go

```diff
@@ -0,0 +1,15 @@
+package security
+
+import "golang.org/x/crypto/bcrypt"
+
+// BcryptPassword returns a hashed token using bcrypt and an error.
+func BcryptPassword(password string) (string, error) {
```

Comment:
With the advent of modern cryptography, the use of bcrypt is deprecated over other hashes like argon2 and scrypt (https://libsodium.gitbook.io/doc/password_hashing).

Reply:
Thanks for the tip, @mdjunior! I have just added Argon2 hash logic here: 60d213a